pom.xml

```diff
@@ -34,10 +33,11 @@
     </developers>
 
     <scm>
-        <url>https://github.com/sinch/sinch-sdk-java/tree/main</url>
-        <connection>scm:git:git://github.com/sinch/sinch-sdk-java.git</connection>
-        <developerConnection>scm:git:ssh://github.com:sinch/sinch-sdk-java.git</developerConnection>
-    </scm>
+        <url>https://github.com/sinch/sinch-sdk-java.git</url>
+        <connection>scm:git:${project.scm.url}</connection>
+        <developerConnection>scm:git:${project.scm.url}</developerConnection>
+      <tag>HEAD</tag>
```

Comment:
Indentation missing

Reply:
The <tag> element is added by maven plugin, so not sure indentation stay synchronized with further action runs